--- nodejs-microtime-0.2.0/index.js~	2011-11-07 20:47:19.000000000 +0200
+++ nodejs-microtime-0.2.0/index.js	2012-06-17 18:24:15.300783077 +0300
@@ -5,8 +5,7 @@
 // `node@0.4.x` and `node@0.6.x`.
 // (from https://github.com/indexzero/daemon.node)
 //
-try { binding = require('./build/default/microtime') }
-catch (ex) { binding = require('./build/Release/microtime') }
+binding = require('./microtime')
 
 exports.now = binding.now
 exports.nowDouble = binding.nowDouble
